schemas/interface.ts

```diff
@@ -183,4 +183,74 @@ declare namespace Quizoot {
          */
         expected_output: string;
     }
+
+    /**
+     * A Quiz object represents a set of questions.
+     *
+     * @version 0.1.0
+     */
+    export interface Quiz {
+        /**
+         * Id of the quiz.
+         */
+        id: string;
+        /**
+         * The title (header) of the quiz.
+         * @minLength 1
+         * @maxLength 40
```

Comment:
Isn't this too restrictive?

Reply:
The max length is not definitive. But I think there should be one, to avoid having too long titles, and keep more control on the rendering (mainly less text ellipsis).